packages/create-playwright/assets/examples/playwright.config.ts

```diff
@@ -0,0 +1,23 @@
+import { PlaywrightTestConfig, devices } from '@playwright/test';
+import path from 'path';
+
+// Reference: https://playwright.dev/docs/test-configuration
+const config: PlaywrightTestConfig = {
+  testDir: path.join(__dirname, '{{testDir}}'),
+
+  // Run your local dev server before starting the tests:
+  // https://playwright.dev/docs/test-advanced#launching-a-development-web-server-during-the-tests
+  webServer: {
+    command: '{{webServerCommand}}',
+    port: 4345,
+  },
+  projects: [
```

Comment:
You don't need projects if there is just one. I'd also not use a device then, but pass `browserName: 'chromium', channel: 'chrome'` instead.